node/core/provisioner/src/lib.rs

```diff
@@ -485,7 +485,7 @@ fn bitfields_indicate_availability(
 	let availability_len = availability.len();
 
 	for bitfield in bitfields {
-		let validator_idx = bitfield.validator_index() as usize;
+		let validator_idx = bitfield.validator_index().0 as usize;
```

Comment:
Adding a `fn as_usize()` might reduce the visual friction a bit :upside_down_face:

Reply:
Ideally this should not be necessary soon at all, once we have https://github.com/paritytech/polkadot/issues/2403 . Which I intend to look into shortly as soon as  the more pressing issues are resolved.